packages/flutter/lib/src/material/dialog.dart

```diff
@@ -951,6 +952,68 @@ class _AdaptiveAlertDialog extends AlertDialog {
   }
 }
 
+/// A widget representing an action button for an alert dialog.
+///
+/// The [AdaptiveAlertAction] class is a stateless widget used to create action buttons
+/// for alert dialogs or similar user interface components. It takes a [child] parameter
+/// to specify the button's content and an [onPressed] callback to define the action when
+/// the button is pressed.
+///
+/// Example usage:
+///
+/// ```dart
+/// AdaptiveAlertAction(
+///   child: Text('OK'),
```

Comment:
The linter tests are complaining about this line. They still check example code, and the `Text` widget needs to be a const.
```suggestion
///   child: const Text('OK'),
```

Reply:
Added `const` against the Text widget in example code.